test/php/library/Icinga/Application/CommonPasswordPolicyTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Tests\Icinga\Application;
+
+use Icinga\Application\Hook\PasswordPolicyHook;
+use Icinga\Application\ProvidedHook\CommonPasswordPolicy;
+use Icinga\Test\BaseTestCase;
+
+class CommonPasswordPolicyTest extends BaseTestCase
+{
+    private object $object;
```

Comment:
Please use the interface `PasswordPolicyHook` instead of `object` as data type.
This ensures that $object is always an instance of `PasswordPolicyHook` and nothing else.

For example, it will automatically throw an error if you try to initialise $object with a class that does not implement `PasswordPolicyHook`.

For example it will throw an error automatically, it you try to initialize $object with a class, which does not implement the `PasswordPolicyHook`.

Try:
```php
public function setUp(): void
{
    $this->object = new \stdClass();
}
```

Reply:
And name the property `$instance` instead of `$object`, as this fits better.